bifrost.go

```diff
@@ -77,6 +78,8 @@ func NewRainbowBridge(bc *BridgeConfig) (RainbowBridge, error) {
 		return newSimpleStorageService(bc)
 	case "gcs":
 		return newGoogleCloudStorage(bc)
+	case "pinata":
+		return newPinataIPFS(bc)
```

Comment:
Sorry, I think the right name is `newPinataCloud`

From `https://pinata.cloud`

If the domain was just  `https://pinata.com` then the name would have been `newPinata`